test/run_test.py

```diff
@@ -232,7 +233,7 @@ def create_local_config(change_repo_dir):
 
 # instead of task.start() (a few missing lines, should be added?)
 cmdline = ["retrace-server-worker", "%d" % task.get_taskid(), "--foreground"]
-call(cmdline)
+run(cmdline, stdout=PIPE, stderr=PIPE, encoding='utf-8')
```

Comment:
Why `stdout/stderr/encoding` when there is nothing done with the output afterwards?